--- a/WudCompress/main.cpp
+++ b/WudCompress/main.cpp
@@ -138 +138 @@ bool compressWUD(wud_t* inputFile, FILE* outputFile, char* outputPath)
-	long long offsetIndexTable = _ftelli64(outputFile);
+	long long offsetIndexTable = ftello(outputFile);
@@ -144 +144 @@ bool compressWUD(wud_t* inputFile, FILE* outputFile, char* outputPath)
-	_fseeki64(outputFile, offsetSectorArrayStart, SEEK_SET);
+	fseeko(outputFile, offsetSectorArrayStart, SEEK_SET);
@@ -194 +194 @@ bool compressWUD(wud_t* inputFile, FILE* outputFile, char* outputPath)
-	_fseeki64(outputFile, offsetIndexTable, SEEK_SET);
+	fseeko(outputFile, offsetIndexTable, SEEK_SET);
@@ -252 +252 @@ int main(int argc, char *argv[])
-		if( stricmp(argv[i], "-noverify") == 0 )
+		if( strcasecmp(argv[i], "-noverify") == 0 )
--- a/WudCompress/wud.cpp
+++ b/WudCompress/wud.cpp
@@ -8,4 +8,4 @@ long long wud_getFileSize64(FILE* file)
-	long long prevSeek = _ftelli64(file);
-	_fseeki64(file, 0, SEEK_END);
-	long long fileSize = _ftelli64(file);
-	_fseeki64(file, prevSeek, SEEK_SET);
+	long long prevSeek = ftello(file);
+	fseeko(file, 0, SEEK_END);
+	long long fileSize = ftello(file);
+	fseeko(file, prevSeek, SEEK_SET);
@@ -17 +17 @@ long long wud_getCurrentSeek64(FILE* file)
-	long long currentSeek = _ftelli64(file);
+	long long currentSeek = ftello(file);
@@ -23 +23 @@ void wud_setCurrentSeek64(FILE* file, long long newSeek)
-	_fseeki64(file, newSeek, SEEK_SET);
+	fseeko(file, newSeek, SEEK_SET);
